app/controllers/api/v1/users_controller.rb

```diff
@@ -5,18 +5,20 @@ module V1
     class UsersController < ApplicationController
       def create
         @user = User.new(user_params)
-        if @user.save
-          render json: @user, status: :created
-        else
-          render json: @user.errors, status: :unprocessable_entity
-        end
+        @user.role_id = Role.find_or_create_by!(name: 'invitado', description: 'invitado').id
+        @user.save!
+        render json: serialize_user, status: :created
```

Comment:
Genial!!! , un detalle nomás:
```suggestion
        @user.role = Role.find_or_create_by!(name: 'invitado', description: 'invitado')
        @user.save!
        render json: serialize_user, status: :created
```